CleverPush/Source/CleverPush.h

```diff
@@ -126,6 +126,7 @@ extern NSString * const CLEVERPUSH_SDK_VERSION;
 + (void)getAppBanners:(NSString*)channelId callback:(void(^)(NSMutableArray <CPAppBanner*>*))callback;
 + (void)getAppBannersByGroup:(NSString*)groupId callback:(void(^)(NSMutableArray <CPAppBanner*>*))callback;
 + (void)setApiEndpoint:(NSString*)apiEndpoint;
++ (void)setAuthorizerToken:(NSString*)authorizerToken;
```

Comment:
@cpKevinrodriguesnobre Any suggestions for the method name, do you think `setAuthorizerToken` is good enough?

Reply:
setAuthorizerToken or setAPIAuthorizerToken
should be okay